php/public/forms.js

```diff
@@ -39,6 +39,15 @@
     document.getElementById('overlay').classList.remove('loading');
   }
 
+  function showPassword(id) {
+    let passwordField = document.getElementById(id);
+    if (passwordField.value !== "" && passwordField.type !== "text") {
```

Comment:
```suggestion
    if (passwordField.type === "password" && passwordField.value !== "") {
```

Reply:
and why?

Reply:
because it is more performant and I think the logic is more understandable by so